Misanthropic-Mistletoes/reactive-nutshell
# Sp edit message

# Description

Edit Messages feature is working.

Fixes https://github.com/Misanthropic-Mistletoes/reactive-nutshell/issues/24

## Type of change

# Testing Instructions
1) `git fetch --all`
2) `git checkout mj-filestructure`
3) Check to see that Messages edit feature is functioning and is updating your entry in your database.json


# Checklist:
